.github/workflows/release_candidate_linux.yml

```diff
@@ -0,0 +1,55 @@
+name: Linux Release Candidate
+
+on: 
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: "Release Candidate Version"
+        required: true
+
+jobs:
+  create_release_candidiate:
+    name: "[${{ github.event.inputs.version_number }}] Linux Release Candidate"
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    environment: Release Candidate / Linux
+    env:
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 0
+
+      - name: Create Release Candidate Tag
+        run: echo "SEMVER_RC_TAG=$(bash ./scripts/semantic_version.sh ${{ github.event.inputs.version_number }} linux prerelease)" >> $GITHUB_ENV
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+
+      - name: Install NPM Dependencies
+        run: npm ci
+
+      - name: Tag Release Candidate
+        run: |
+          git config --global user.name "OutlineBot"
+          git config --global user.email "firehook-team-admin@google.com"
+          git tag ${{ env.SEMVER_RC_TAG }}
+
+      - name: Build Linux Release Candidate
+        run: npm run action src/electron/release_candidate_linux ${{ env.SEMVER_RC_TAG }}
```

Comment:
alternatively I suppose we could bundle this with just the version number so it could be re-used. however if someone happened to get their hands on it we'd have no easy way to verify it wasn't a production build (it being unsigned, I guess?)

Reply:
I think I answered my own question here, so I'm putting this back.